update known issues (#355)
appsignal/appsignal-docs: source/support/known-issues/puma-phased-restart.html.md

---

> title: Puma phased restart

Reviewer: ```suggestion
title: Puma phased restarts
```

---

## Affected components

> - AppSignal for Ruby gem versions: `v2.9.17` (see [workarounds](#workaround)).

Reviewer: ```suggestion
- AppSignal for Ruby gem version `v2.9.17` and later (see [workarounds](#workaround)).
```

## Description

We've added a new plugin to add our [Puma magic dashboard](https://docs.appsignal.com/ruby/integrations/puma.html#minutely-probe) in version `2.9.17` of our gem, which broke phased restarts.

## Symptoms

> - Puma phased restart are broken.

Reviewer: ```suggestion
- The AppSignal integration stops, but is not restarted when using Puma phased restarts.
```

---
title: Puma phased restarts
---

## Affected components

- AppSignal for Ruby gem version `v2.9.17` and later (see [workarounds](#workaround)).

## Description

> We've added a new plugin to add our [Puma magic dashboard](https://docs.appsignal.com/ruby/integrations/puma.html#minutely-probe) in version `2.9.17` of our gem, which broke phased restarts.

Reviewer 2: ```suggestion
We've added a new plugin to add our [Puma magic dashboard](https://docs.appsignal.com/ruby/integrations/puma.html#minutely-probe) in version `2.9.17` of our gem, which breaks Puma phased restarts.
```

> See issue [#575](https://github.com/appsignal/appsignal-ruby/issues/575) on the appsignal-ruby repo for more details on the specific error.

Reviewer 2: ```suggestion
See [appsignal-ruby issue #575](https://github.com/appsignal/appsignal-ruby/issues/575) for more details on the specific error.
```

More descriptive link label.

## Symptoms

The AppSignal integration stops, but is not restarted when using Puma phased restarts.

## Workaround

> Locking to `2.9.16` should fix that issue for now.

Reviewer 2: ```suggestion
- Downgrade the AppSignal gem to `2.9.16` until the issue has been resolved.
- Do not use Puma's phased restart feature until the issue has been resolved.
```
